src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -373,15 +373,20 @@ void Chttp2ServerListener::ActiveConnection::HandshakingState::OnHandshakeDone(
   HandshakingState* self = static_cast<HandshakingState*>(args->user_data);
   OrphanablePtr<HandshakingState> handshaking_state_ref;
   RefCountedPtr<HandshakeManager> handshake_mgr;
-  bool cleanup_connection = false;
+  bool remove_connection = false;
+  OrphanablePtr<ActiveConnection> connection;
```

Comment:
Not sure why this was moved. It's functionally no different AFAICT, and it moves the variable declaration further away from "as close to the first use as possible". https://google.github.io/styleguide/cppguide.html#Local_Variables